pkg/controller/containersource/reconcile.go

```diff
@@ -50,6 +51,8 @@ type reconciler struct {
 	recorder      record.EventRecorder
 }
 
+const tryTargetable = true
```

Comment:
comment here might be helpful. I assume it means that we first try it as sinkable, then targetable?

Reply:
added a comment.